.wordlist.txt

```diff
@@ -212,7 +214,7 @@ PiSupply
 PiVPN
 Plex
 pre
-ProFTP
+preferenced
```

Comment:
`preferenced` does not exist, it is `preferred` 😉.

Reply:
Yep, changed.

Reply:
I am actually working on the File Servers subpage within "Optimized Software". Then the actual spell error "ProFTP" instead of "ProFTPD" will disappear.  
Wait a couple of days for this. :-)

Reply:
Yes, `ProFTPD` is fully correct for the server daemon that is installed.